src/viewer.js

```diff
@@ -373,9 +389,43 @@ $.Viewer = function( options ) {
         if (!_this._updateRequestId) {
             _this._updateRequestId = scheduleUpdate( _this, updateMulti );
         }
+
+        var tiledImage = event.item;
+        var fullyLoadedHandler =  function() {
+            var newFullyLoaded = _this._areAllFullyLoaded();
+            if (newFullyLoaded !== _this._fullyLoaded) {
+                _this._fullyLoaded = newFullyLoaded;
+                _this.raiseEvent('fully-loaded-change', { fullyLoaded: newFullyLoaded });
```

Comment:
This is where we need a doc comment, to document the existence of the `fully-loaded-change` event for the Viewer. You can see the TiledImage version here: 

https://github.com/openseadragon/openseadragon/blob/393ab25d74949f8895298330d96cebaaa255ec31/src/tiledimage.js#L268